apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/RequestResponsePopulateMdcFilter.java

```diff
@@ -96,15 +96,15 @@ else if (headerValues.size() == 1)
 
   /**
    * @param keySuffix the suffix to build a full key for
-   * @return the key to use for {@link MDC#put(String, String)}
+   * @return the key to use for {@link BfdMDC#put(String, String)}
    */
   private static String computeMdcKey(String keySuffix) {
```

Comment:
can we pull this method and the one below up into the shared utilities?

Reply:
I think we can, yeah. There are two of these, and they might handle things differently. I'll try it out and see.